packages/vite/src/node/plugins/html.ts

```diff
@@ -141,6 +141,17 @@ export const assetAttrsConfig: Record<string, string[]> = {
   use: ['xlink:href', 'href'],
 }
 
+// Some `<link rel>` elements should not be inlined in build. Excluding:
+// - `shortcut icon`                : deprecated, use `icon` only
```

Comment:
```suggestion
// - `icon`                : deprecated, use `icon` only
```
I think this is more correct.

Reply:
No I think he meant to list deprecated rel name like "shortcut icon", the comment explain why the list doesn't contains "shortcut icon"

Reply:
My bad, I meant ``// - `shortcut`                : only valid for IE < 9, use `icon` ``.